### DISCUSSION TITLE
Use on multiple accounts [Frequently Asked]

### DISCUSSION THREAD

**Discussion by d0biar** [Refloow/Steam-Auto-Friend#7] [Q&A] [answered]
"How to use multiple accounts"
"Add option to use multiple accounts"
"How can i use it on 100 accounts"
"How to use more then 1 account"

**Comment by Refloow** [comment, marked as answer]
### **Support for multiple accounts wont be added to the script.**
 **_Point of the script is accepting and auto responding to messages on active account that is in use._**

You can run the script on multiple accounts by making copies of the script, but script is built to work on one and only one account, even tho you could copy and paste the code or run the script multiple times. We wont provide an integrated way to do so unless you know how to code, which we cant stop you for making any modifications, its legal to do so, but keep in mind that responsibility of using or modifying the software is yours.

**Adding modifications that make you gain any type of advantage or modifications for usage of community market, causing the spam on the platform in any way might make the platform to take actions.**

Feel free to checkout our [User Agreement](https://github.com/OSL-Works/Steam-Auto-Friend/blob/master/.github/USER_AGREEMENT.md)
And project's [open source license](https://github.com/OSL-Works/Steam-Auto-Friend/blob/master/LICENSE)
